tests/pytests/test_language.py

```diff
@@ -0,0 +1,144 @@
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'SORTBY', 'word', 'DESC')
+    expected = [2, '{word}:1', ['word', 'arancia'], '{word}:2', ['word', 'arance']]
+    env.assertEqual(res, expected)
+
+    # Search for "arancia" using English
+    # should return 1 results using invalid stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'language', 'english')
+    env.assertEqual(res, [1, '{word}:1', ['word', 'arancia']])
+
+    # Search for "oranges" using English
+    # should return 2 results using stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'english',
+                  'SORTBY', 'word', 'ASC')
+    expected = [2, '{word}:3', ['word', 'orange'], '{word}:4', ['word', 'oranges']]
+    env.assertEqual(res, expected)
+
+    # Search for "orange" in Italian
+    # should return 1 results using invalid stemming in Italian
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'italian')
+    env.assertEqual(res, [1, '{word}:4', ['word', 'oranges']])
+
+    # drop index
+    env.cmd('FT.DROP', 'idx')
+
+
+def testSearchJsonIndexLanguage(env):
+    conn = getConnectionByEnv(env)
+
+    # Create index - language Italian by default
+    env.cmd('FT.CREATE', 'idx', 'ON', 'JSON',
+            'LANGUAGE', 'italian', 'SCHEMA', '$.word', 'AS', 'word', 'TEXT')
+    
+    # Create sample data in Italian
+    conn.execute_command('JSON.SET', '{word}:1', '$', '{"word":"arancia"}') # orange
+    conn.execute_command('JSON.SET', '{word}:2', '$', '{"word":"arance"}') # oranges
+
+    # Create sample data in English
+    conn.execute_command('JSON.SET', '{word}:3', '$', '{"word":"orange"}') # orange
+    conn.execute_command('JSON.SET', '{word}:4', '$', '{"word":"oranges"}') # oranges
+
+    # Wait for index to be created
+    waitForIndex(env, 'idx')
+
+    # Search for "arancia", should use the language by default: Italian
+    # It should return 2 results using stemming in Italian
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'SORTBY', 'word', 'DESC', 
+                  'NOCONTENT')
+    env.assertEqual(res, [2, '{word}:1', '{word}:2'])
+
+    # Search for "arancia" using English
+    # It should return 1 results using invalid stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'arancia', 'language', 'english')
+    env.assertEqual(res, [1, '{word}:1', ['$', '{"word":"arancia"}']])
+
+    # Search for "oranges" using English
+    # It should return 2 results using stemming in English
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'english',
+                  'SORTBY', 'word', 'ASC', 'NOCONTENT')
+    env.assertEqual(res, [2, '{word}:3', '{word}:4'])
+
+    # Search for "orange" using Italian
+    # It should return 1 results using invalid stemming in Italian
+    res = env.cmd('FT.SEARCH', 'idx', 'oranges', 'language', 'italian')
+    env.assertEqual(res, [1, '{word}:4', ['$', '{"word":"oranges"}']])
+
+    # drop index
+    env.cmd('FT.DROP', 'idx')
```

Comment:
Why is this needed? Please remove if not

Reply:
I removed it.